src/Form/FastlySettingsForm.php

```diff
@@ -141,6 +141,15 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       ],
     ];
 
+    $webhook_url = count($form_state->getValues()) ? $form_state->getValue('webhook_url') : $config->get('webhook_url');
```

Comment:
It might be worth adding a checkbox for enable/disabled status of the web hook so this can be easy toggle. This is useful if you want to turn off notifications for dev/stage/ci/local environments.